packages/mongodb-chatbot-evaluation/testData/expected_links_tests.yaml

```diff
@@ -0,0 +1,30 @@
+- name: Search for 'and'
+  tags:
+    - top_search_term
+  messages:
+    - role: user
+      content: and
+  expectation: "N/A"
+  expectedLinks:
+    - reference/operator/query/and
+    - reference/operator/aggregation/and
+    - reference/operator/query/all
+- name: Search for 'or'
+  tags:
+    - top_search_term
+  messages:
+    - role: user
+      content: or
+  expectation: "N/A"
+  expectedLinks:
+    - reference/operator/query/or
+    - reference/operator/aggregation/or
+    - reference/operator/query-logical
+- name: Search for 'not'
+  tags:
+    - top_search_term
+  messages:
+    - role: user
+      content: not
+  expectation: "N/A"
+  # no expected links
```

Comment:
Could we allow `expectedLinks: null` or `expectedLinks: []` to explicitly define this use case? I think we'd have to refine our implementation slightly.

Reply:
yeah we can make the construct a little more flexible.

Reply:
i did!

Reply:
I guess in this case I'd expect to see `expectedLinks: []` so that we're explicitly saying "there should be no related links" as opposed to "we're not testing for related links"